calico_containers/integrations/kubernetes/calico_kubernetes.py

```diff
@@ -260,14 +261,26 @@ def _generate_rules(self):
         inbound_rules = [
             {
                 'action': 'allow',
-            },
+            }
         ]
 
         outbound_rules = [
             {
                 'action': 'allow',
-            },
+            }
         ]
+
+        print('Getting Policy Rules from Annotation of pod %s' % pod)
+
+        annotations = self._get_annotations(pod)
+        namespace = self._get_namespace(pod)
+
+        if 'allowFrom' in annotations.keys():
+            inbound_rules = eval(annotations['allowFrom'])
```

Comment:
I wouldn't use the eval() builtin to load json - it could do other things that we don't want it to (e.g. someone could embed python in their annotation and we'd start executing it.)

Take a look at https://docs.python.org/2/library/json.html